wca/logger.py

```diff
@@ -25,6 +28,38 @@
 
 log = logging.getLogger(__name__)
 
+_module_record_counters = defaultdict(Counter)
+
+
+class CountingHandler(logging.Handler):
+    def __init__(self, counters):
+        super().__init__()
+        self.counters = counters
```

Comment:
Check if passed argument is valid with:
``` from wca.config import assure_type ```
```suggestion
        assure_type(counters, Dict[str, Counter])
        self.counters = counters
```